maths/fibonacci_sequence_memoization.py

```diff
@@ -0,0 +1,29 @@
+cache: dict[int, int] = {}
```

Comment:
Prefill the cache...
```suggestion
cache: dict[int, int] = {1: 1, 2: 1}
```